pusher/package.json

```diff
@@ -40,6 +40,8 @@
   },
   "homepage": "https://github.com/thecodingmachine/workadventure#readme",
   "dependencies": {
+    "@types/cookie": "^0.4.1",
```

Comment:
all @types/smth should be under `devDependencies` please

Reply:
Makes sense! I've changed that!